core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpAuthConnectionContextTest.java

```diff
@@ -168,6 +168,28 @@ public void testGoodAuthentication() throws Exception {
         });
     }
 
+    @Test
+    public void testGoodAuthenticationWithExtraData() throws Exception {
+        runInAuthContext(() -> {
+            try {
+                boolean authSequenceCompleted = authenticate(AUTH_KEY_ID1, AUTH_PRIVATE_KEY1,
+                        "weather,location=us-midwest temperature=82 1465839830100400200\n");
+                Assert.assertTrue(authSequenceCompleted);
+            } catch (RuntimeException ex) {
+                // Expected that Java 8 does not have SHA256withECDSAinP1363
+                if (ex.getCause() instanceof NoSuchAlgorithmException && TestUtils.getJavaVersion() <= 8) {
```

Comment:
@glasstiger: on what Java you were getting this error? Was it in the CI? 
I tried it on Azul Zulu 8u332 and it does have the algorithm.